templates/files/apps/aws/aws-ebs-csi-driver-app.yaml

```diff
@@ -9,7 +9,7 @@ data:
       replicas: 2
       hostNetwork: false
       nodeSelector:
-        kubernetes.io/os: linux
+        kubernetes.io/os: master
```

Comment:
no this is plain wrong. just remove the `nodeSelector` field completely. The default will be used which is to run on master nodes